src/controls/placeholder/PlaceholderComponent.tsx

```diff
@@ -120,9 +120,10 @@ export class Placeholder extends React.Component<IPlaceholderProps, IPlaceholder
           const iconTextClassNames = `${styles.placeholderText} ${(this.state.width && this.state.width <= 380) ? styles.hide : ""}`;
           const iconTextEl = typeof iconText === 'string' ? <span className={iconTextClassNames}>{this.props.iconText}</span> : iconText(iconTextClassNames);
           const descriptionEl = typeof description === 'string' ? <span className={styles.placeholderDescriptionText}>{this.props.description}</span> : description(styles.placeholderDescriptionText);
+          const editMode = document.location.href.toLocaleLowerCase().indexOf('mode=edit') !== -1;
 
           return (
-            <div className={`${styles.placeholder} ${this.props.contentClassName ? this.props.contentClassName : ''}`} ref={this._linkElm}>
+            <div className={`${styles.placeholder} ${this.props.contentClassName ? this.props.contentClassName : ''} ${this.props.hideInReadMode && !editMode ? styles.hide : ''}`} ref={this._linkElm}>
```

Comment:
instead of hiding the element and all the children, can we just return an empty element instead if in read mode? Not sure if this would work, but if it does, seems like a cleaner approach as we avoid adding elements to the page that we know will be always hidden